browser.js

```diff
@@ -310,6 +310,11 @@ document.addEventListener('DOMContentLoaded', () => {
 	// enable OS specific styles
 	document.documentElement.classList.add(`os-${process.platform}`);
 
-	// TODO: figure out a better way to detect when React is done
-	setTimeout(init, 300);
+	// detect when React is ready before firing init
+	const awaitReact = setInterval(() => {
```

Comment:
Good call. Do you want to have it support arrays/NodeLists or just individual nodes?